src/app.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+import { Header } from './header';
+import { Main } from './main';
+import { Footer } from './footer';
+
+import { Error } from './error';
+
+import  "./app.scss";
+
+export const App = (
+	<div>
+		< Header />
```

Comment:
do not use space after opening bracket 
`<Header />`
`<Error...`
`< Main />`
etc